source/vir/src/triggers_auto.rs

```diff
@@ -195,8 +205,63 @@ fn trigger_var_depth(ctxt: &Ctxt, term: &Term, depth: u64) -> Option<u64> {
     }
 }
 
-fn make_score(term: &Term, depth: u64) -> Score {
-    Score { depth, size: term_size(term) }
+fn make_score(term: &Term, depth: u64, num_operators: u64) -> Score {
+    Score { depth, size: term_size(term), num_operators }
+}
+
+fn count_bit_operators(ctx: &Ctx, exp: &Exp) -> u64 {
```

Comment:
This will be much simpler if it just counts the `BitOp` terms in `Term` (similar to how `term_size` works) rather than traversing the `Exp`.

Reply:
I see, that's a really better way. Thanks!